R/merge_performers.r

```diff
@@ -1,27 +1,30 @@
 #' @title Merge Performers
 #' @description Apply annotations to performers in spek.
 #' @details Adds newly found performers and updates esiting performers in spek
-#' @import jsonlite
+# #' @import jsonlite
```

Comment:
Is this import being commented out?  If so, shouldn't it be replaced with `importFrom jsonlite fromJSON toJSON` ?